src/lib/containers/IconSvg.md

```diff
@@ -4,7 +4,6 @@
       options={{
         icon: 'data',
         color: '#e61899',
-        size: 'sm',
```

Comment:
Delete the size option because material ui icons will use the surrounding font size to size the icon. But I might put it back later if we need to customize the icon size for bigger or smaller than surrounding fonts.